src/Styling/Home.css

```diff
@@ -109,6 +109,7 @@
   color: #fff;
   border: 1px solid #fff;
   cursor: pointer;
+  position: relative;
```

Comment:
Why does this button need a position of relative?

Reply:
Removed z-index and replaced it with position relative. z-index is no longer used and bug issue was resolved.